it/config/DocumentationControllerISpec.scala

```diff
@@ -80,37 +81,42 @@ class DocumentationControllerISpec extends IntegrationBaseSpec {
   }
 
   "an OAS documentation request for V2" must {
```

Comment:
@SamuelJacobsHMRC Can we remove "for V2" from the description of the test name as now it tests both V2 & V3

Reply:
Updated